script/check-st2-ok

```diff
@@ -51,9 +52,31 @@ if [ -x "${ST2_PATH}" ]; then
     echo "First time starting this machine up?"
     echo "Visit https://${ST2_IP}/setup to configure StackStorm"
     echo "Otherwise, head to https://${ST2_FQDN} to access the WebUI"
-    echo ""
-    echo "If you would like to use the CLI interface, please log out and"
-    echo "login to your session again. This is necessary to properly source"
-    echo "StackStorm ENV variables"
+    echo
+    echo "If you would like to use the CLI interface, you will need"
+    echo "to ensure that StackStorm environment variables are properly"
+    echo "set. You can do this by logging out and logging back in, or"
+    echo "running the command:"
+    echo
+    echo ". /etc/profile.d/st2.sh"
+    echo
+    echo "Thanks for installing StackStorm! Come visit us in our Slack Channel"
+    echo "and tell us how it's going. We'd love to hear from you!"
+    echo "http://stackstorm.com/community-signup"
+}
+
+# Only run the check for StackStorm in the event that it actually exists
```

Comment:
So, we're saying that it is probably ok if StackStorm doesn't exist on the box...